src/components/UI/Button/Button.module.scss

```diff
@@ -28,4 +30,12 @@
   svg {
     margin-right: 10px;
   }
+
+  &[class~='wallet-btn'] {
```

Comment:
Please change to scss syntax

Reply:
writing:    &.wallet-btn   doesn't seem to work.

Reply:
1.It won't work - CSS modules add id to each class at runtime. You should use the `styles` object of the CSS file imported to the component. i.e., `styles.walletBtn`.
2. Why are you adding app-specific class component styles (wallet-btn) to a generic component styles?